README.md

```diff
@@ -1,49 +1,100 @@
-# Fresh SEO 🍋
 
-The fastest way ⚡️ to create sitemaps in your Deno
-[Fresh project](https://fresh.deno.dev/) project 🍋
+# Fresh SEO 🍋   [![Badge License]][License]
 
-## Getting started
+*The fastest way to create sitemaps* <br>
+*for your Deno **[Fresh project]**.*
```

Comment:
Could we remove the extra newline and italic usage for text? It makes it a little harder to read imo

Reply:
Sure~